x11-wm/pekwm/pekwm-0.2.1.ebuild

```
@@ -0,0 +1,86 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib desktop optfeature
+
+DESCRIPTION="A lightweight window manager initially based on aewm++"
+HOMEPAGE="
+	https://www.pekwm.se/
+	https://github.com/pekdon/pekwm
+"
+SRC_URI="
+	https://github.com/pekdon/${PN}/releases/download/release-${PV}/pekwm-${PV}.tar.gz
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="debug +jpeg +png truetype +xft xinerama +xpm"
+
+RDEPEND="
+	png? ( media-libs/libpng:0 )
+	jpeg? ( virtual/jpeg:0 )
+	xpm? ( x11-libs/libXpm )
+	x11-libs/libX11
+	x11-libs/libXext
+	truetype? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )"
+
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DENABLE_IMAGE_JPEG=$(usex jpeg)
+		-DENABLE_IMAGE_PNG=$(usex png)
+		-DENABLE_IMAGE_XPM=$(usex xpm)
+		-DENABLE_XINERAMA=$(usex xinerama)
+		-DENABLE_XFT=$(usex truetype)
+	)
+
+	cmake_src_configure
+}
+
+multilib_src_install() {
+
+	cmake_src_install
+
+	# Install contributor scripts into doc folder
+	docinto contrib
+	dodoc "${S}"/contrib/lobo/*.{pl,vars,png} "${S}"/contrib/lobo/README
+
+	# Insert an Xsession
+exeinto /etc/X11/Sessions
+newexe - pekwm <<- _EOF_
+	#!/bin/sh
+	/usr/bin/pekwm
+_EOF_
```

```suggestion
	# Insert an Xsession
	exeinto /etc/X11/Sessions
	newexe - pekwm <<- _EOF_
		#!/bin/sh
		/usr/bin/pekwm
	_EOF_
```
Don't forget to indent.

---

OK, but I only commited your previous suggestion, I assumed that this is OK for code inserts.